cmd/image-builder/config.go

```diff
@@ -141,11 +142,11 @@ func (gitState GitStateConfig) IsPullRequest() bool {
 
 func LoadGitStateConfig(ciSystem CISystem) (GitStateConfig, error) {
 	switch ciSystem {
-	// Load from env specific for github actions
+	// Load from env specific for Github Actions
 	case GithubActions:
 		return loadGithubActionsGitState()
-	// Load from env specific for prow jobs
-	case Prow:
+	// Load from env specific for Prow Jobs and Azure DevOps
+	case Prow, AzureDevOps:
 		return loadProwJobGitState()
```

Comment:
```suggestion
		return loadADOGitState()
```
Prow will be shutdown. Should we change function name to the specific for ADO as it will stay. See line 157.

Reply:
renamed